episodes/transfer_data.md

```diff
@@ -29,13 +29,71 @@ Objectives
 
 ## Exercises
 
-::::{dropdown} Challenge Title
+::::{dropdown} What should you store in your home directory?
 :open:
 
-Challenge content
+Select all that apply:
+
+- [ ] Notebooks
+- [ ] Code
+- [ ] Large and/or shared datasets with read-write permissions
+- [ ] Text files
+- [ ] Small datasets
+- [ ] Configuration files
+- [ ] Read-only datasets, e.g for teaching
+- [ ] Temporary data, e.g. intermediate results from a calculation
+
+:::{dropdown} Solution
+
+- [x] Notebooks
+- [x] Code
+- [ ] Large and/or shared datasets with read-write permissions – *consider using Cloud Object Storage or other cloud-native storage solutions*
+- [x] Text files
+- [x] Small datasets
+- [x] Configuration files
+- [ ] Read-only datasets, e.g for teaching – *consider placing these files in the `~/shared` directory*
+- [ ] Temporary data, e.g. intermediate results from a calculation – *consider using the `/tmp` directory*
+
+:::
+
+::::
+
+:::{dropdown} Data requirements
+:open:
+
+Think about the data that you or your community require for your/their work.
+
+- What type of data is it? E.g. tabular data (.csv, .xls), images (.png, .jpg, .bmp), other scientific data formats (.h5, .nc, .fastq)
+- How large is the dataset?
+- Where is the data stored?
+- Is the same dataset used by a group of collaborators?
+
+Write up your thoughts in our shared collaborative document.
+
+:::
+
+::::{dropdown} Push GitHub code
+:open:
+
+You use the Git JupyterLab extension to push code to a remote repository on GitHub and the following box opens:
+
+:::{image} ../media/episodes/transfer_data/gh-credentials.png
+:alt: Information box asking for GitHub credentials such as username and Personal Access Token.
+:width: 80%
+:align: center
+
+:::  
+
+<br>
+
+What do you do?
+
+1. Enter your GitHub username and Personal Access Token as requested, then click *Save my login temporarily*
+1. Click *Cancel*, open a Terminal, enter the command `gh-scoped-creds` and follow the prompt
 
 :::{dropdown} Solution
-Solution content
+
+The correct answer is 2.  
```

Comment:
Can you expand this solution a bit? Include a short explanation of why option 1 is not recommended?

Reply:
Resolved in https://github.com/czi-catalystproject/hub-champion-training/pull/28/commits/3b10899e13683e06eefdce4aeed950ac95f928a8